jsonrpc-specs/src/methods/discv5.json

```diff
@@ -0,0 +1,568 @@
+[
+	{
+		"name": "discv5_nodeInfo",
```

Comment:
This would probably be a great method to start with for testing.  All of the clients should probably have some way to initialize them with a known private key for their ENR record.  This would allow a simple test that 1) initializes the node with a known private key and then 2) hits this endpoint to check that the returned ENR matches the expected value.